applications/services/desktop/desktop_i.h

```diff
@@ -56,42 +55,37 @@ struct Desktop {
     DesktopMainView* main_view;
     DesktopViewPinTimeout* pin_timeout_view;
     DesktopSlideshowView* slideshow_view;
+    DesktopViewPinInput* pin_input_view;
 
     ViewStack* main_view_stack;
     ViewStack* locked_view_stack;
 
-    DesktopSettings settings;
-    DesktopViewPinInput* pin_input_view;
-
     ViewPort* lock_icon_viewport;
     ViewPort* dummy_mode_icon_viewport;
     ViewPort* clock_viewport;
     ViewPort* stealth_mode_icon_viewport;
 
-    AnimationManager* animation_manager;
-
     Loader* loader;
     NotificationApp* notification;
 
-    FuriPubSubSubscription* app_start_stop_subscription;
+    FuriPubSub* status_pubsub;
     FuriPubSub* input_events_pubsub;
     FuriPubSubSubscription* input_events_subscription;
+
     FuriTimer* auto_lock_timer;
     FuriTimer* update_clock_timer;
 
-    FuriPubSub* status_pubsub;
+    AnimationManager* animation_manager;
+    FuriSemaphore* animation_semaphore;
 
     DesktopClock clock;
+    DesktopSettings settings;
 
-    bool in_transition : 1;
-    bool locked : 1;
-
-    FuriSemaphore* animation_semaphore;
+    bool in_transition;
```

Comment:
not a big thing, but why no longer packed bitfields?

Reply:
Doesn't matter due to mandatory 4 byte padding in the end of the struct